src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -122,13 +122,10 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
             </FormControl>
           </Grid>
         </Grid>
-        <Box display="flex" gap={2} sx={{ mt: 2 }}>
-          <SubmitButton variant="contained" color="primary" type="submit" disabled={!allowSubmit}>
-            Create
-          </SubmitButton>
-          <Button variant="outlined" color="secondary" onClick={onCancel}>
-            Cancel
-          </Button>
+        <Box textAlign="right" sx={{ my: 2 }}>
+          <NERSuccessButton variant="contained" type="submit" sx={{ mx: 1 }}>
```

Comment:
Just in case you forget, but remember that the cancel button should be to the left of the 'submit' button.